renovate.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
this file must go to a different PR

Reply:
Was a mistake with rebasing, it's gone now.